Services/Data/default.json

```diff
@@ -63,12 +63,13 @@
   ],
   "Rules": [
     {
-      "Id": "default_Chiller_Pressure_High",
       "Name": "Chiller pressure too high",
       "Enabled": true,
       "Description": "Pressure > 250",
-      "GroupId": "default_Chillers",
```

Comment:
Can you revert the GroupId values to what we had before prefix of `default_`?

Reply:
should we change this for telemetry service too?